var/spack/repos/builtin/packages/hip/package.py

```diff
@@ -82,7 +82,10 @@ def get_rocm_prefix_info(self):
                           for name in ('llvm-amdgpu', 'hsa-rocr-dev',
                                        'rocminfo', 'rocm-device-libs'))
             mydict['rocm-path'] = self.spec.prefix
-            device_lib_path = mydict['rocm-device-libs'].amdgcn.bitcode
+            if '@:3.8.0' in self.spec:
+                device_lib_path = mydict['rocm-device-libs'].lib
+            else:
+                device_lib_path = mydict['rocm-device-libs'].amdgcn.bitcode
```

Comment:
I don't this this works at all, does it? Shouldn't it be:
```suggestion
            if '@:3.8.0' in self.spec:
                device_lib_path = mydict['rocm-device-libs'].prefix.lib
            else:
                device_lib_path = mydict['rocm-device-libs'].prefix.amdgcn.bitcode
```

Reply:
@adamjstewart , the prefix is already appended in the L81 ..mydict = dict((name, self.spec[name].prefix).